manifests/site.pp

```diff
@@ -68,7 +68,17 @@
   include role::pkg
 }
 
-# Jenkins controller for ci.jenkins.io
+node 'oracle.updates.jenkins.io' {
+  sshkeyman::hostkey { ['oracle.updates.jenkins.io']: }
+  # volume configuration is in hiera
```

Comment:
Can you add more detail to where is this hiera more specifically?
As it is I'm not sure

Reply:
not possible: it depends of the hiera's loading hierarchy.

It can be at different levels in hieradata:

- node (usually hieradata/*/clients but could be a postgresql database, etcd store, etc.)
- role
- profile
- etc.

Since it is a key feature of Puppet, the goal of this comment is to "trigger" the reader so they can look to Puppet's doc or search the repo :)

Reply:
WDYT about adding a link to https://forge.puppet.com/modules/puppetlabs/lvm/0.3.2 ? (alas I can't find any anchor in the 0.3.2 page to give a link )?